src/features/edit/js/gridEdit.js

```diff
@@ -112,12 +112,45 @@
               }
             },
             methods: {
-              edit: { }
+              edit: {
+                /**
+                 * @ngDoc method
+                 * @name editCell
+                 * @description invokes a cells editor and optionally allows the edit to persist
+                 * <pre>
+                 *      gridApi.edit.editCell(row, col, true);
+                 * </pre>
+                 * @param {integer} row of the cell to edit
+                 * @param {integer} col of the cell to edit
+                 * @param {boolean} persist the cell in edit mode. Prevent cancel events such as scroll from removing the editor
+                 */
+                editCell: function (row, col, persist) {
+                  if (persist) {
+                    this.rows[row].cols[col].$scope.persistEdit = true;
+                  }
+                  this.rows[row].cols[col].$scope.$broadcast('editCell');
+                },
+                /**
+                 * @ngDoc method
+                 * @name editEditCell
```

Comment:
You need a @methodOf param here.
Example:
https://github.com/angular-ui/ui-grid/blob/master/src/features/cellnav/js/cellnav.js#L418